go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/influxdata/usage-client v0.0.0-20160829180054-6d3895376368
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/lestrrat-go/jwx/v2 v2.0.19
-	github.com/microcosm-cc/bluemonday v1.0.16
+	github.com/microcosm-cc/bluemonday v1.0.26
```

Comment:
This was done as part of the 1.21.8 update; can you give more detail why it was updated?

Reply:
The version 1.0.16 was retract by author:

```
go get ./...   
go: warning: github.com/microcosm-cc/bluemonday@v1.0.16: retracted by module author: Retract older versions as only latest is to be depended upon
go: to switch to the latest unretracted version, run:
        go get github.com/microcosm-cc/bluemonday@latest
```